src/sql/workbench/browser/modal/dialogHelper.ts

```diff
@@ -6,50 +6,38 @@
 'use strict';
 import { SelectBox } from 'sql/base/browser/ui/selectBox/selectBox';
 import { Button } from 'sql/base/browser/ui/button/button';
+import { append, $, addClass, addClasses } from 'vs/base/browser/dom';
 
-import { Builder } from 'sql/base/browser/builder';
 import * as types from 'vs/base/common/types';
 
 import * as azdata from 'azdata';
 
-export function appendRow(container: Builder, label: string, labelClass: string, cellContainerClass: string, rowContainerClass?: string): Builder {
-	let cellContainer: Builder;
-	let rowAttributes = rowContainerClass ? { class: rowContainerClass } : {};
-	container.element('tr', rowAttributes, (rowContainer) => {
-		rowContainer.element('td', { class: labelClass }, (labelCellContainer) => {
-			labelCellContainer.div({}, (labelContainer) => {
-				labelContainer.text(label);
-			});
-		});
-		rowContainer.element('td', { class: cellContainerClass }, (inputCellContainer) => {
-			cellContainer = inputCellContainer;
-		});
-	});
+export function appendRow(container: HTMLElement, label: string, labelClass: string, cellContainerClass: string, rowContainerClass?: string | Array<string>): HTMLElement {
```

Comment:
Could you please verify this doesn't change CSS classes that breaks style where this code is used?  Some of our styles are pretty specific about the required DOM hierarchy.

Reply:
I had to make this change because I found a bug with my initial changes.